addons/repo-proxy/common/src/main/java/org/commonjava/indy/repo/proxy/ContentBrowseRemoteIndyListingRewriteManager.java

```diff
@@ -78,8 +84,8 @@ public void initIndyClient()
             try
             {
                 //FIXME: Here we used a MemPassMgr for simple. Maybe some changes in future for more complex cases.
-                indyClient =
-                        new Indy( siteConfig, new MemoryPasswordManager(), mapper, new IndyContentBrowseClientModule() );
+                indyClient = new Indy( siteConfig, new MemoryPasswordManager(), mapper,
```

Comment:
There's an Indy-specific password manager implementation IIRC, but I think you're right...we should never have this path locked down, so password managers should never really matter too much here. If we had a no-op password manager, that would work too I think.

Reply:
Ok. Let's just make it KISS now.